### PR TITLE
Clarify TTL calculation with comment

### PR DESCRIPTION
**Pull request by matthpeterson**

### CODE REVIEW

**Review by jschmidt10 — DISMISSED** [review]

**Review by apmoriarty — DISMISSED** [review]
LGTM

File: warehouse/core/src/main/java/datawave/iterators/filter/TokenizingFilterBase.java [hunk 1]
```diff
@@ -101,11 +101,16 @@ public boolean accept(AgeOffPeriod period, Key k, Value V) {
...
             return true;
         }
+        // cutoffTimestamp includes the default TTL
         long cutoffTimestamp = period.getCutOffMilliseconds();
         
-        // if ttl exists, subtract it from the cut off timestamp
+        // If there is a TTL for this key:
         if (calculatedTTL > 0) {
-            cutoffTimestamp -= calculatedTTL - period.getTtl() * period.getTtlUnitsFactor();
+            // cutoffTimestamp is currently offset by the default TTL. Start by undoing that default offset
```
Comment (ivakegg) [inline warehouse/core/src/main/java/datawave/iterators/filter/TokenizingFilterBase.java:109]: Excellent.  good job

File: warehouse/core/src/test/java/datawave/iterators/filter/ColumnVisibilityLabeledFilter.java [hunk 2]
```diff
@@ -101,7 +101,9 @@ public boolean accept(AgeOffPeriod ageOffPeriod, Key k, Value V) {
                 long cutOff = ageOffPeriod.getCutOffMilliseconds();
                 // move cut-off back by the timeToLive
                 if (timeToLive > 0) {
+                    // deduct TTL for this key
```
Comment (ivakegg) [inline warehouse/core/src/test/java/datawave/iterators/filter/ColumnVisibilityLabeledFilter.java]: Just a thought, but should we switch these two statements to be consistent with the other class?

**Review by ivakegg — CHANGES_REQUESTED** [review]

File: warehouse/core/src/test/java/datawave/iterators/filter/ColumnVisibilityLabeledFilter.java (hunk 2 above)
Comment (matthpeterson) [reply warehouse/core/src/test/java/datawave/iterators/filter/ColumnVisibilityLabeledFilter.java]: ok will do

**Review by ivakegg — APPROVED** [review]

**Review by apmoriarty — APPROVED** [review]

**Pull request merged**